src/libpmempool/replica.c

```diff
@@ -165,6 +195,31 @@ create_replica_health_status(struct pool_set *set, unsigned repn)
 	return replica_hs;
 }
 
+/*
+ * replica_part_remove_recovery_file -- remove bad blocks recovery file
+ */
+void
+replica_part_remove_recovery_file(struct part_health_status *phs)
+{
+	LOG(3, "phs %p", phs);
+
+	if (phs->recovery_file == NULL || phs->exist_recovery_file == 0)
```

Comment:
can `exist_recovery_file == 1 && recovery_file == NULL` ? what would happen then? Do we pass NULL to os_unlink?